inspire/modules/workflows/tasks/submission.py

```diff
@@ -91,6 +91,7 @@ def _create_ticket(obj, eng):
             email=email,
             title=d.title,
             identifier=sip.metadata.get("system_number_external", {}).get("value", ""),
+            user_comment=sip.metadata.get("hidden_note", {}).get("value", ""),
```

Comment:
I'd actually do it like
```
user_comment=filter(lambda k: 'submitter' == k['source'],
                    sip.metadata.get("hidden_note", []))
```

So this way if there isn't a user comment (a dict value in the `metadata['hidden_note']` list with the `source` key as `submitter`), then `user_comment=[]` and in the jinja template will skip it.